sql/common/src/main/scala/org/apache/sedona/sql/UDF/Catalog.scala

```diff
@@ -178,14 +179,10 @@ object Catalog {
     function[RS_Array](),
     function[RS_Normalize](),
     function[RS_Append](),
-    function[RS_AddBandFromArray](),
```

Comment:
Why remove other functions?

Reply:
This was an accident while merging with master.
I have reverted this